src/Histogram.hpp

```diff
@@ -40,11 +39,16 @@ class Histogram: private std::vector<T> {
     /// Get the size of the bins
     double bin_size() const {return dr_;}
 
+    /// Get the min
+    double min() const {return min_;}
+
     /// Insert some `data` in the histogram, and guess the position using the
     /// `bin_size` of the Histogram.
     void insert(T new_data) {
```

Comment:
Doc is out of date. Can you rename `new_data` to `data` and remove the part about `bin_size`?